backend/src/utils/valuesOrNull.js

```diff
@@ -0,0 +1,13 @@
+const valuesOrNull = (...values) => values.reduce((prev, curr) => {
```

Comment:
Maybe that's just a preference. But I like when function names contain verbs.

Reply:
Also using reduce is not really efficient. It doesn't really matter performance-wise but the code is more complex.
Fors are not so bad if they do the job better than array operators.
 ```javascript
for(const arg of args) {
  if(arg !== undefined) {
    return arg
  }
}
return null
```